inference-engine/src/transformations/include/transformations/convert_opset1_to_legacy/convert_mul_or_add_finally.hpp

```diff
@@ -35,6 +35,7 @@ class ngraph::pass::ConvertMulOrAddFinally: public ngraph::pass::GraphRewrite {
     // This pass finally converts single Multiply and Add operations to ScaleShift or Power operation
     ConvertMulOrAddFinally() : GraphRewrite() {
```

Comment:
Since it is not only MulOrAdd conversion, please rename it to ConvertLinOpFinally